packages/volto/locales/ca/LC_MESSAGES/volto.po

```diff
@@ -1729,6 +1729,11 @@ msgstr "Si esteu segur que aquest usuari ha abandonat l'objecte, podeu desbloque
 msgid "If you are certain you have the correct web address but are encountering an error, please contact the {site_admin}."
 msgstr "Si esteu segur que teniu l'adreça web correcta però trobeu un error, poseu-vos en contacte amb el {site_admin}."
 
+#. Default: "If you have only one sort on a property, please place it in the top sort field."
+#: components/manage/Blocks/Search/schema
+msgid "If you have only one sort on a property, please place it in the top sort field."
```

Comment:
I don't understand this message. And if I don't understand it, then an editor won't either. It needs to use language that the end user will understand.

In the UI, `property` is not used. Under `Controls`, I see `Sort on options`. Additionally if there is only one option selected, it is already at the top.

Finally, this should be a drag and drop field to allow the user to reorder the selection of sort on options, without having to remove them, then add them back.

I think this is the intent:

```suggestion
msgid "Select the options by which you want to sort items. Sorting by multiple options will be performed in order from first to last selected options."
```

If that's not correct, please elaborate.

Reply:
@stevepiercy We have two sort on field in search
 block see screenshots. For second one I want to add a description that if they only have only one sorton options then please select the value in upper sorton field.

<img width="293" alt="Screenshot 2024-07-19 at 8 25 14 PM" src="https://github.com/user-attachments/assets/1a1d33dc-5e76-43e4-b4c1-8e92c82e4889">

Reply:
I don't think you understand what these two options actually do. See https://6.docs.plone.org/volto/user-manual/blocks.html#search-block. If that does not help, please ask for clarification.

Ultimately I think a new message is not necessary. However perhaps other elements of the UI could be improved to make their function clearer to newcomers.